install-dev/data/db_structure.sql

```diff
@@ -2273,6 +2273,10 @@ CREATE TABLE `PREFIX_log` (
   `message` text NOT NULL,
   `object_type` varchar(32) DEFAULT NULL,
   `object_id` int(10) unsigned DEFAULT NULL,
+  `id_shop` int(10) unsigned DEFAULT NULL,
+  `id_shop_group` int(10) unsigned DEFAULT NULL,
+  `id_lang` int(10) unsigned DEFAULT NULL,
+  `in_all_shop` tinyint(1) unsigned NOT NULL DEFAULT '0',
```

Comment:
@PululuK I'm not sure about the naming here, of course it is accurate but it should be `in_all_shops`, right? I was thinking about something like `is_global_context`? but `in_all_shops` seems to be more clear 🤔

Reply:
@kpodemski  Thanks !
Q : In what context was the action performed?
R: `in_all_shops`

WDYF ?

Reply:
@PululuK yes, `in_all_shops` is ok :)